java-frontend/src/main/java/org/sonar/java/se/ExplodedGraph.java

```diff
@@ -69,7 +69,7 @@ public Node node(ProgramPoint programPoint, @Nullable ProgramState programState)
 
     private final Map<Node, Edge> edges = new HashMap<>();
 
-    boolean isNew;
+    public boolean isNew;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make isNew a static final constant or non-public and provide accessors if needed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)